ui/partials/main/thInfraMenu.html

```diff
@@ -30,10 +30,10 @@
      target="_blank">HgStats</a>
 </li>
 <li>
-  <a href="http://status.taskcluster.net/"
+  <a href="https://status.taskcluster.net/"
```

Comment:
This URL times out. The new location is https://tools.taskcluster.net/status/ and supports HTTPS :-)